vocabularies/voc4cat/0007231.ttl

```diff
@@ -5,11 +5,13 @@ PREFIX xsd: <http://www.w3.org/2001/XMLSchema#>
 <https://w3id.org/nfdi4cat/voc4cat_0007231>
     a skos:Concept ;
     dcterms:identifier "voc4cat_0007231"^^xsd:token ;
+    skos:altLabel
+        "becker"@en ,
+        "beker"@en ;
+    skos:broader <https://w3id.org/nfdi4cat/voc4cat_0007250> ;
     skos:definition "In laboratory equipment, a beaker (also becker or beker) is generally a cylindrical container with a flat bottom."@en ;
     skos:historyNote "0009-0008-1278-8890 Michael Götte created the resource"@en ;
```

Comment:
- Please remove alt. labels from definition text.
- Should we list these alt.labels at all? I have never heard them and the only source is wikipedia but in a quick search I could not find any use of the terms beker/becker in science/engineering.

Reply:
I added them as suggestion from @nmoust

Reply:
i removed them from the text, i think we can leave them as alt labels